muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/pedagogy/PedagogyController.java

```diff
@@ -253,7 +253,22 @@ public PedagogyForm findFormByStudentIdentifier(String studentIdentifier) {
   }
   
   public void createViewHistory(PedagogyForm form, Long modifierId) {
+    List<PedagogyFormHistory> historyList = pedagogyFormHistoryDAO.listByFormAndCreator(form, modifierId);
+    
+    if (historyList != null) {
```

Comment:
Technically not needed, as DAO methods are guaranteed to always return an empty list.